src/components/GameView/GameOverDialog.vue

```diff
@@ -41,33 +41,51 @@
           </div>
         </div>
       </template>
+      <p v-if="opponentWantsToRematch">{{ opponent.username }} wants to rematch</p>
     </template>
 
     <template #actions>
-      <v-btn
-        color="surface-1"
-        variant="flat"
-        data-cy="gameover-go-home"
-        :loading="leavingGame"
-        @click="goHome"
-      >
-        Go Home
-      </v-btn>
+      <div class="d-flex gap-2">
+        <v-btn
+          class="mr-4"
+          color="surface-1"
+          variant="flat"
+          data-cy="gameover-go-home"
+          :loading="leavingGame"
+          @click="goHome"
+        >
+          Go Home
+        </v-btn>
+        <v-btn
+          color="primary"
+          variant="flat"
+          data-cy="gameover-rematch"
+          @click="rematch"
+        > Rematch </v-btn>
```

Comment:
Let's do the newlines like on the first button
```suggestion
          @click="rematch"
        >
          Rematch
        </v-btn>
```

Reply:
Linter doesn't allow me to put it in new lines